src/main/java/daomephsta/unpick/api/classresolvers/IClassResolver.java

```diff
@@ -21,6 +22,24 @@ public interface IClassResolver {
 	@Nullable
 	ClassReader resolveClass(String internalName);
```

Comment:
I think a simpler solution would be to simply change this to return `ClassNode`. We are still in beta, and tbh this was the last breaking change I wanted to make before the release.

Reply:
Updated